nautobot/core/tests/test_api.py

```diff
@@ -18,13 +18,16 @@
 from nautobot.core import testing
 from nautobot.core.api.parsers import NautobotCSVParser
 from nautobot.core.api.renderers import NautobotCSVRenderer
+from nautobot.core.api.utils import get_serializer_for_model
 from nautobot.core.api.versioning import NautobotAPIVersioning
 from nautobot.core.constants import COMPOSITE_KEY_SEPARATOR
+from nautobot.core.utils.lookup import get_route_for_model
 from nautobot.dcim import models as dcim_models
 from nautobot.dcim.api import serializers as dcim_serializers
 from nautobot.extras import choices, models as extras_models
 from nautobot.ipam import models as ipam_models
 from nautobot.ipam.api import serializers as ipam_serializers
+from nautobot.tenancy.models import TenantGroup
```

Comment:
Would be good to follow the established pattern here, i.e.:
```suggestion
from nautobot.tenancy import models as tenancy_models
```

and use `tenancy_models.TenantGroup` below.